### PR TITLE
Search bundle corrections

### PR DESCRIPTION
**Pull request by jochenmanz**
### 1. Why is this change necessary?
better developer experience

### 2. What does this change do, exactly?
fixes some incorrect phpdoc type hints

### 6. Checklist

- [ ] I have written tests and verified that they fail without my change
- [X] I have squashed any insignificant commits
- [ ] This change has comments for package types, values, functions, and non-obvious lines of code
- [X] I have read the contribution requirements and fulfil them.

### CODE REVIEW

File: engine/Shopware/Bundle/SearchBundleDBAL/VariantHelper.php [hunk 1]
```diff
@@ -45,7 +45,7 @@ class VariantHelper implements VariantHelperInterface
     protected $connection;
 
     /**
-     * @var CustomFacetGatewayInterface
+     * @var CustomListingHydrator
      */
     protected $customFacetGateway;
```
Comment (mitelg) [inline engine/Shopware/Bundle/SearchBundleDBAL/VariantHelper.php:50]: how about a renaming? :blush:

Comment (teiling88) [reply engine/Shopware/Bundle/SearchBundleDBAL/VariantHelper.php:50]: It would be a breaking change in my opinion.

Comment (mitelg) [reply engine/Shopware/Bundle/SearchBundleDBAL/VariantHelper.php:50]: wasn't the visibility changed a few days ago? of course this is a BC, but would be ok imho, if this change was made in the RC

Comment (jochenmanz) [reply engine/Shopware/Bundle/SearchBundleDBAL/VariantHelper.php:50]: not according to git blame. Should i include this change in the new PR?

Comment (mitelg) [reply engine/Shopware/Bundle/SearchBundleDBAL/VariantHelper.php:50]: that were other properties. so I think we have to stick with the name

**Comment by jochenmanz** [comment]
merged into the new pr https://github.com/shopware/shopware/pull/1792

**Pull request closed without merging**